docs/running/cloud.rst

```diff
@@ -0,0 +1,288 @@
+allows us to spin up a cluster without any external dependencies using the Toil
+Appliance, a Docker image that bundles Toil and all its requirements, e.g.
+Mesos. Toil will automatically choose an appliance image that matches the
+current Toil version but that choice can be overriden by setting the
+environment variables ``TOIL_DOCKER_REGISTRY`` and ``TOIL_DOCKER_NAME`` or
+``TOIL_APPLIANCE_SELF`` (see :func:`toil.applianceSelf` and
+:ref:`appliance_dev` for details)::
+
+    toil launch-cluster -p aws CLUSTER-NAME-HERE \
+         --nodeType=t2.micro \
+         --keyPairName=your-AWS-key-pair-name
+
+to launch a t2.micro leader instance -- adjust this instance type accordingly
+to do real work. See `here <https://aws.amazon.com/ec2/instance-types/>`_ for a
+full selection of EC2 instance types. For more information on cluster
+management using Toil's AWS provisioner, see :ref:`clusterRef`.
+
+To use CGCloud-based autoscaling, see :ref:`installationAWS` for CGCloud
+installation and more information on starting our leader instance.
+
+Once we have our leader instance launched, the steps for both provisioners
+converge. As with all distributed AWS workflows, we start our Toil run using an
+AWS job store and being sure to pass ``--batchSystem=mesos``. Additionally, we
+have to pass the following autoscaling specific options. You can read the help
+strings for all of the possible Toil flags by passing ``--help`` to your toil
+script invocation. Indicate your provisioner choice via the
+``--provisioner=<>`` flag and node type for your worker nodes via
+``--nodeType=<>``. Additionally, both provisioners support `preemptable nodes
+<https://aws.amazon.com/ec2/spot/>`_. Toil can run on a heterogenous cluster of
+both preemptable and non-preemptable nodes. Our preemptable node type can be
+set by using the ``--preemptableNodeType=<>`` flag. While individual jobs can
+each explicitly specify whether or not they should be run on preemptable nodes
+via the boolean `preemptable` resource requirement, the
+``--defaultPreemptable`` flag will allow jobs without a `preemptable`
+requirement to run on preemptable machines. Finally, we can set the maximum
+number of preemptable and non-preemptable nodes via the flags ``--maxNodes=<>``
+and ``--maxPreemptableNodes=<>``. Insure that these choices won't cause a hang
+in your workflow - if the workflow requires preemptable nodes set
+``--maxPreemptableNodes`` to some non-zero value and if any job requires
+non-preemptable nodes set ``--maxNodes`` to some non-zero value. If the
+provisioner can't provision the correct type of node for the workflow's jobs,
+the workflow will hang. Use the ``--preemptableCompensation`` flag to handle
+cases where preemptable nodes may not be available but are required for your
+workflow.
+
+.. admonition:: Using a shared filesystem
+
+   Toil currently only supports a ``tempdir`` set to a local, non-shared directory.
```

Comment:
This isn't a concern with autoscaling, this probably fits better back in the section about running on shared file systems.